serverless.yml

```diff
@@ -268,6 +268,31 @@ functions:
               - name: request.querystring.search_after
               - name: request.header.origin
 
+  # This get_token will get tokens from the elasticsearch index
+  es_get_token_handler:
+    handler: handlers/token_api.get_token
+    timeout: 15
+    maximumRetryAttempts: 0
+    package:
+      patterns:
+        - 'handlers/token_api.py'
+    layers:
+      - { Ref: PythonRequirementsLambdaLayer }
+    events:
+      - http:
+          path: tokens/{token_id}
+          method: get
+          cors: true
+          caching:
+            enabled: true
+            ttlInSeconds: 300
+            cacheKeyParameters:
+              - name: request.querystring.search_text
```

Comment:
Do we need these? I thought they were used only by the `get_tokens` API, not the get one token.

Reply:
Removed the unused `cacheKeyParameters`